.github/workflows/nuget_slack_notifications.yml

```diff
@@ -95,6 +95,10 @@ jobs:
                  rabbitmq.client
                  restsharp
                  serilog
+                 serilog.extensions.logging
+                 serilog.aspnetcore
```

Comment:
Aren't these secondary dependencies, rather than things we instrument?

Reply:
They are, but when they update, I think there's a chance our instrumentation could be broken even if indirectly. `Serilog.Extensions.Logging` seems like one we definitely need to keep up with. The sinks I could go either way, but unless someone remembers to update the sinks while applying other Serilog updates, we'll fall behind again.

Reply:
We can try it :)